src/client/components/login-component/index.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+import {Link} from 'react-router-dom';
+const React = require('react');
+const LoginForm = require('../login-form/index.js');
+
+class LoginComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'status': 'being',
+      'error': null,
+    };
+  }
+
+  handleLoginResponse(status, response) {
+    if (status === 400) {
+      this.handleFieldsError(response);
+    } else if (status === 403) {
+      this.handleMisMatch(response);
+    } else if (status === 500) {
+      this.handleUnknownError(response);
+    } else if (status === 200) {
+      window.location.href = '/feed';
+    }
+  }
+
+  handleFieldsError(response) {
+    let error = JSON.parse(response).errorType;
+    let errorMessage;
+    if (error === 'ContentType') {
+      errorMessage = 'Something went wrong, please try later!';
+      this.setState({'error': errorMessage});
+    } else if (error === 'FieldMissing') {
+      errorMessage = 'Don\'t try to hack our site!';
+      this.setState({'error': errorMessage});
+    } else if (error === 'WrongFormat') {
+      errorMessage = 'Don\'t try to hack our site!';
+      this.setState({'error': errorMessage});
+    }
+  }
+
+  handleMisMatch(response) {
+    if (response.errorType === 'MisMatch') {
+      let errorMessage = 'Username and password mismatch!';
+      this.setState({'error': errorMessage});
+    }
+  }
+
+  handleUnknownError(response) {
+    if (response.errorType === 'Unknown') {
+      let errorMessage = 'Unknown Error!';
+      this.setState({'error': errorMessage});
+    }
+  }
+
+  collectData(event) {
+    return {
+      'username': event.target.elements.namedItem('username').value,
+      'password': event.target.elements.namedItem('password').value,
+    };
+  }
+
+  // validateData(data) {
+  //   let reg = /[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,3}$/;
+  //   if (!reg.test(data.username)) {
+  //     this.setState({'isUsernameValid': true});
+  //   }
+  // }
+
+  sendData(data) {
+    let xhr = new XMLHttpRequest();
+    xhr.addEventListener('readystatechange', function() {
+      if (xhr.readyState === XMLHttpRequest.DONE) {
+        this.handleLoginResponse(xhr.status, xhr.response);
+      }
+    }.bind(this));
+    xhr.open('POST', '/api/login');
+    xhr.setRequestHeader('Accept', 'application/json');
+    xhr.setRequestHeader('Content-Type', 'application/json');
+    this.setState({'status': 'loading'});
+    xhr.send(JSON.stringify(data));
+  }
+
+  submitLogin(event) {
+    event.preventDefault();
+    let LoginInfo = this.collectData(event);
```

Comment:
it is just a simple variable, you dont need to capitalize this variable. use loginInfo instead